backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+  name               String               @db.VarChar
+  jurisdictionId     String               @map("jurisdiction_id") @db.Uuid
+  jurisdictions      Jurisdictions        @relation(fields: [jurisdictionId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  amiChartItem       AmiChartItem[]
+  unitGroupAmiLevels UnitGroupAmiLevels[]
+  units              Units[]
+
+  @@map("ami_chart")
+}
+
+// Note: [birth_month, birth_day, birth_year] formerly max length 8; [phone_number, phone_number_type] formerly max length 16; 
+//       [first_name, middle_name, last_name] formerly max length 64
+// Note: [first_name, last_name] formerly min length 1
+// Note: [email_address] needs to have lower case enforcement
+model Applicant {
+  id                   String        @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt            DateTime      @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt            DateTime      @updatedAt @map("updated_at") @db.Timestamp(6)
+  firstName            String?       @map("first_name")
+  middleName           String?       @map("middle_name")
+  lastName             String?       @map("last_name")
+  birthMonth           String?       @map("birth_month")
+  birthDay             String?       @map("birth_day")
+  birthYear            String?       @map("birth_year")
+  emailAddress         String?       @map("email_address")
+  noEmail              Boolean?      @map("no_email")
+  phoneNumber          String?       @map("phone_number")
+  phoneNumberType      String?       @map("phone_number_type")
+  noPhone              Boolean?      @map("no_phone")
+  workInRegion         YesNoEnum?    @map("work_in_region")
+  workAddressId        String?       @unique() @map("work_address_id") @db.Uuid
+  addressId            String?       @unique() @map("address_id") @db.Uuid
+  applicantWorkAddress Address?      @relation("applicant_work_address", fields: [workAddressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  applicantAddress     Address?      @relation("applicant_address", fields: [addressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  applications         Applications?
+
+  @@map("applicant")
+}
+
+// Note: [rule] used to be a different enum but prisma doesn't support that kind of enum yet. See: https://github.com/prisma/prisma/issues/273
+model ApplicationFlaggedSet {
+  id                    String               @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt             DateTime             @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt             DateTime             @updatedAt @map("updated_at") @db.Timestamp(6)
+  rule                  RuleEnum
+  ruleKey               String               @unique() @map("rule_key") @db.VarChar
+  resolvedTime          DateTime?            @map("resolved_time") @db.Timestamptz(6)
+  listingId             String               @map("listing_id") @db.Uuid
```

Comment:
do we need listingId with the listings relation?

Reply:
yeah so
```
  listingId             String               @map("listing_id") @db.Uuid
listings              Listings             @relation(fields: [listingId], references: [id], onDelete: NoAction, onUpdate: NoAction)
```

listingId in this case is used by that listings relation to explain to prisma how that relationship is connected (hopefully that makes sense)